.ebextensions/tomcatlogs.config

```diff
@@ -0,0 +1,30 @@
+###################################################################################################
+#### The configuration below sets the logs to be pushed, the Log Group name to push the logs to and
+#### the Log Stream name as the instance id. The following files are examples of logs that will be
+#### streamed to CloudWatch Logs in near real time:
+####
+#### /var/log/tomcat8/catalina.out
+####
+#### You can then access the CloudWatch Logs by accessing the AWS CloudWatch Console and clicking
+#### the "Logs" link on the left. The Log Group name will follow this format:
+####
+#### /aws/elasticbeanstalk/<environment name>/<full log name path>
+####
+#### Please note this configuration can be used additionally to the "Log Streaming" feature:
+#### http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/AWSHowTo.cloudwatchlogs.html
+###################################################################################################
+
+files:
+  "/etc/awslogs/config/tomcatlogs.conf" :
+    mode: "000644"
+    owner: root
+    group: root
+    content: |
+      [/var/log/tomcat8/catalina.out]
+      log_group_name=`{"Fn::Join":["/", ["/aws/elasticbeanstalk", {"Ref":"AWS::StackName"}, "var/log/tomcat8/catalina.out"]]}`
+      log_stream_name={instance_id}
```

Comment:
Would it be helpful to have an identifier in addition to the instance_id here? Like `synapse-login-scipoolprod-instance_id`?

Reply:
I don't think that's necessary because the `log_stream_name` is placed under a path.  it is placed under the `log_group_name` therefore it looks like this..
```
/aws/elasticbeanstalk/awseb-e-2ec22bbasb-stack/var/log/tomcat8/catalina.out
  -- i-0917b89b624ffb7d1
  -- i-a12317ba9b624fa12
```